### PR TITLE
Minor update to conngen documentation

### PR DESCRIPTION
**Pull request by sarakonradi**
This PR removes the extra backtick and updates the wording.

### CODE REVIEW

File: doc/guides/conngen_interface.rst [hunk 1]
```diff
@@ -6,10 +6,9 @@ Since 2014 (`Djurfeldt et al.,
 Connection Generator Interface. This allows to couple connection
 generating libraries to NEST without having to modify NEST itself.
 
-In contrast to the `built-in connection
-functions :doc:`Connection management <connection_management>` and the Topology
-module, the Connection Generator Interface has a different way for specifying
-the connectivity:
+In contrast to the :doc:`Connection management <connection_management>` and
```
Comment (jougs) [inline doc/guides/conngen_interface.rst:9]:
```suggestion
In contrast to the :doc:`regular creation of connections <connection_management>` and
```

File: doc/guides/conngen_interface.rst [hunk 2]
```diff
@@ -6,10 +6,9 @@ Since 2014 (`Djurfeldt et al.,
...
+Topology module's built-in connection functions, the Connection Generator
```
Comment (jougs) [inline doc/guides/conngen_interface.rst:10]: Do we still talk about Topology? Isn't this now part of regular `Connect` somehow? Or should this also include a link to the corresponding manual? Sorry if I missed something.

File: doc/guides/conngen_interface.rst [hunk 3]
```diff
@@ -6,10 +6,9 @@ Since 2014 (`Djurfeldt et al.,
...
+Interface has a different way of specifying connectivity:
```
Comment (jougs) [inline doc/guides/conngen_interface.rst:11]:
```suggestion
In contrast to the :doc:`regular creation of connections <connection_management>, the
Connection Generator Interface provides a different way of specifying connectivity:
```

**Comment by sarakonradi** [comment]
> I've updated my suggestion based on more thinking and the input by @terhorstd.

Done!

**Comment by jougs** [comment]
@sarakonradi: #1623 fixed some Travis build failures. Please `git pull upstream master`.

**Comment by sarakonradi** [comment]
> @sarakonradi: #1623 fixed some Travis build failures. Please git pull upstream master.

Thanks for fixing the build failures! The checks have now passed.

**Pull request merged**